bin/node/runtime/Cargo.toml

```diff
@@ -60,6 +60,7 @@ pallet-im-online = { version = "2.0.0", default-features = false, path = "../../
 pallet-indices = { version = "2.0.0", default-features = false, path = "../../../frame/indices" }
 pallet-identity = { version = "2.0.0", default-features = false, path = "../../../frame/identity" }
 pallet-membership = { version = "2.0.0", default-features = false, path = "../../../frame/membership" }
+pallet-mmr = { version = "2.0.0", default-features = false, path = "../../../frame/merkle-mountain-range" }
```

Comment:
What is `pallet-mmr` responsible for?

Reply:
I've removed this pallet because according to the [documentation](https://paritytech.github.io/substrate/master/pallet_mmr/index.html), this pallet is experimental and has not to be used in production. Polkadot doesn’t use this pallet in [0.8.27](https://github.com/paritytech/polkadot/blob/v0.8.27/runtime/polkadot/Cargo.toml) but [uses](https://github.com/paritytech/polkadot/commit/e8050450b71aef41210f7fa5ec2bca6d494ef610#diff-d257db32f1322827de450e474842896422eb21d757b797fd16273874d70b2a1f) it in older versions in a testnet.
[Link to our testing log](https://www.notion.so/cere/Testing-logs-1f7d5107060a432398bad8b6a15225f0#321a8dd8b7944fcdadff61d3e05e0c10)